wikicurses/main.py

```diff
@@ -35,6 +37,111 @@ def tabComplete(text, matches):
     return match
 
 
+def showImages():
+    """
+    Launch program to display the images for the current article.
+
+    The program is specified in the config, along with it's args.
+    Otherwise, the default is `feh`.
+    """
+    if 'wikipedia.org' not in wiki.siteurl:
+        ex.notify('Cancelled: Only works on Wikipedia')
+        return
```

Comment:
http://vim.wikia.com/wiki/Vim_Tips_Wiki this one? It's on Wikia which uses MediaWiki, I thought.

Anyway, I'll have a go at changing to the API.